### DISCUSSION TITLE
SQL Database Chain - MS SQL Connection String

### DISCUSSION THREAD

**Discussion by JeffCristine** [FlowiseAI/Flowise#848] [Q&A]
What is the proper connect string to connect to a MS SQL server running in a docker container?  It is on the same system as Flowise AI, also running in a docker container.  The SQL server (docker host) IP address is 10.0.0.218 and TCP port is 1401, not 1433 in this case.  

I've tried: 110.0.0.218:1401/DatbaseName

For some reason I seem to need to put another "1" in the IP address?

The response from the "SQL DB Chain" is: "ConnectionError: Failed to connect to 10.0.0.218:1401 - self-signed certificate"

How do I add "trustServerCertificate: true" to the connect string as well as a username / password?

I can't find any documentation on this topic (MS SQL Server Connection Strings).

Thank you,

Jeff Cristine

**Comment by dxforceiwei** [comment]
i have same issue, any answer?

**Reply by JeffCristine** [reply]
No answer at all.  I also posted the question in the FlowiseAI Discord channel (twice):

https://discord.com/channels/1087698854775881778/1147038534117625856

https://discord.com/channels/1087698854775881778/1105430878101962826/threads/1142034664148701184

**Comment by jjamgmv** [comment]
If someone solve the issue, please, publish the information

**Comment by MFMedias57** [comment]
mysql://{user}:{password}@host.docker.internal{port}/{database}
